src/static/riot/search.tag

```diff
@@ -746,16 +746,16 @@
                 <div class="mobile_labelwrap"></div>
                 <span class="participant_label ui right floated mini label tooltip" data-content="Participant count">
                 <i class="user icon"></i> {participant_count}
-            </span>
+                </span>
+                <span style="text-overflow: ellipsis;" class="deadline_label ui right floated red mini label tooltip"
+                      data-content="Deadline of the current phase"
+                      >
+                <i class="alarm icon"></i> {deadline_date(current_phase_deadline)}
```

Comment:
So this issue is for showing deadlines that aren't currently showing. Every competition listed on Chahub should show (in some way):

1. Competition has ended
2. Competition never ends
3. Competition (or phase? not sure how it's setup exactly in the backend) is ending on <date>

Reply:
This logic is done in the deadline_date function. There will be a return of the Deadline alarm tag for every competition.

1. If the competition is < 0 milliseconds from the current date, it will display days since the competition completed.
2. If there is no phase deadline set, "No Phase Deadline" will be on the alarm tag. I can change to "Competition Never Ends" or "Phase Never Ends" if preferred.
3. This is handled in the deadline_date function.

Reply:
Here is a preview of the Deadline Date tag with different times remaining of competitions.
[Preview](https://imgur.com/a/cqmtMaz)